worlds/soe/__init__.py

```diff
@@ -272,6 +306,10 @@ def generate_basic(self):
         wings_item = self.create_item('Wings')
         self.world.get_location(wings_location, self.player).place_locked_item(wings_item)
         self.world.itempool.remove(wings_item)
+        if self.energy_core == EnergyCore.option_vanilla:
+            energy_core = self.create_item('Energy Core')
+            self.world.get_location('Energy Core #285', self.player).place_locked_item(energy_core)
+            self.world.itempool.remove(energy_core)
```

Comment:
removing an item from itempool is pretty expensive, as you might be scanning a list of thousands of entries, not adding the item to the itempool would be better.